migrations/20201001094246-add-is-active-to-user.js

```diff
@@ -0,0 +1,5 @@
+exports.up = (db, callback) => db.addColumn('user', 'isActive', { type: 'boolean', notNull: true, defaultValue: false })
```

Comment:
should the default value here be `true` to match the one in the Sequelize model?

Reply:
i preferred letting the app handle making it true but i updated it to be false in DB ... my opinion isn't super strong on this

Reply:
I would think we would want the application enforcing the same defaults as the database, otherwise things could get confusing